### ISSUE TITLE
switch from File list view and Outline view 

### ISSUE THREAD

**Issue by leewi9** [typora/typora-issues#1243]
Currently it is designed to switch between these two, it's already quite useful, but if i orgnize a big project, got many files, i need to switch betwwen many files and also each outline views, is there any way to make another left panel for FileList so that we don't need to click switch anymore.

![image](https://user-images.githubusercontent.com/10832544/36348621-bc93422a-14ae-11e8-9a9c-f603a412a046.png)

then the outline on the top right can be cancelled.

**Comment by leewi9** [comment]
just to better illustrate my idea, here is the design i want, no switch, just simple and intruititive to use, perhaps with show/hide panel options, that's enough

![image](https://user-images.githubusercontent.com/10832544/36654569-89d52222-1af8-11e8-919b-30822712ab03.png)

**Comment by abnerlee** [comment]
I can understand your requirements.

> but if i orgnize a big project, got many files

Are you using Typora for taking notes or writing a book?

**Comment by leewi9** [comment]
I'm using typora to mange all my personal wiki, including knowledge manage, code snippet, everything i need to take down, so a easy-to-use navigation is very important

**Comment by leewi9** [comment]
@abnerlee yes, will you figure out a better way?

**Comment by leewi9** [comment]
@abnerlee any update commander??

**Comment by abnerlee** [comment]
I understand some user will use Typora as their PKM or note-taking tool, since there may not be other good alternatives. 

But this may not be a high priority issue, so we may consider this after v1.0

**Comment by leewi9** [comment]
you're right about there are no other good alternatives, hope will get update soon, when v1.0?

**Comment by greyli** [comment]
I am using Typora to writing book (thanks for this excellent project), it will be much convenient to display both Files and Outline panel. Look forward to this feature!

**Comment by mli** [comment]
I'm also using Typora to write a book, available at https://d2l.ai/. The books consists of ~100 Jupyter notebooks, but Typora is significantly better editor. Having both files and outline will be very useful.

**Comment by leewi9** [comment]
@abnerlee any update please, hope to see it solved asap

**Comment by Gowa2017** [comment]
Yes, it's a good idea.

**Comment by h4tr3d** [comment]
Seeing Outline along with Files Tree is a good approach. I want it too. 

Also, Is there any plans for the cost politics after 1.0 release?

**Comment by mukul-raina** [comment]
+1 to this feature request

**Comment by ajhutchinson4** [comment]
+1 to this feature request

**Comment by liangchiwuu** [comment]
+1 this would be extremely useful

**Comment by kbore** [comment]
is this feature added?

**Comment by finalcreator** [comment]
+1 this would be extremely useful

**Comment by ff12323** [comment]
+1 I hope so

**Comment by beyond2001** [comment]
+1 Hope this feature is added soon since V1.0 is now released

**Comment by eriksdev1** [comment]
@abnerlee  @beyond2001  I wish this feature can be considered now that I purchased v1 ! amazing note taking app, please consider enabling two panels one for the files and one for the outline...

**Comment by acccoco** [comment]
+1 this would be extremely useful

**Comment by eriksdev1** [comment]
by any chance can there be an update ? this dates from 2018 😲

**Comment by YuanTao1995** [comment]
Hope this useful feature! I want show file tree at LEFT side bar for my personal knowledge management and meanwhile the outline for current file at RIGHT side bar.  @abnerlee

**Comment by CongDeveloper** [comment]
This feature is usefull. Both these two panels are commonly used, since there is enough space, why not display them at the same time? @abnerlee

**Comment by samishchandra** [comment]
This feature would be really useful. Also if simply allowing the sidebar to be placed on the right vs left using custom css is a good option to have. Right now I can't find a css option to place the sidebar to the right.

**Comment by eriksdev1** [comment]
I bought a copy of the program in hopes of someday seeing this feature implemented, maybe there is something preventing from being developed ? Typora is awesome and this could make it a lot more useful and awesome ! Many thanks for this great program.

**Comment by eriksdev1** [comment]
Hello, I found something that might provide inspiration, I found a program named Deepdwn, this is the layout that works for me and maybe many of us
![image](https://user-images.githubusercontent.com/45124393/173330780-c30a9555-450f-4659-ac25-66669417b24f.png)

**Comment by finalcreator** [comment]
+1 Hope

**Comment by ff12323** [comment]
create a cmd file, open one path at multi windows
`
"C:\Program Files\Typora\Typora.exe" C:\Users\Administrator\Desktop\\notes
`

**Comment by ldeng-ustc** [comment]
+1, need this

**Comment by tmdautov** [comment]
Do you happen to have any updates on this issue? Or how can I stick the outline with a custom CSS theme? 

The sticky outline is a significant feature when we work with large Wiki files.

<img width="1332" alt="image" src="https://user-images.githubusercontent.com/6027337/212523301-5c649f81-49a5-4d54-8114-f7fab793fc93.png">

**Comment by sospiess** [comment]
+1

I found the discussion on this issue by also looking for a way to switch the sidebar from left to right. Especially when attaching a larger screen to my Laptop it is a tat distracting to have the folders/outline panel right in the middle between whatever I want to take notes on (web article/meeting window) and the acutal editor.
Thanks for this great product. :)

**Comment by GXFG** [comment]
+1, need this @2023

**Comment by megaobanana** [comment]
+1, Very looking forward on this feature

**Comment by Lawrence-of-AnKing** [comment]
+1, I need this for a loong time, at least 5 years or longer, noting changed from beta vision to 1.7.5...

**Comment by shijianzqbx** [comment]
+1,This is very necessary

**Comment by eriksdev1** [comment]
@abnerlee it feels real bad to not know, if this is not happening please let us know :'(

**Comment by heylukegit** [comment]
I switched to obsidian for a while and switched back because Typora has way nicer live preview. 
The only thing I miss from Obsidian is being able to have both folder tree and content outline in two separate sidebars. 
Please consider add the feature. Thanks!
@abnerlee

**Comment by champbob** [comment]
+1
This would be quite nice to have, as I also like to jump between files to reference things in my notes before going back to my currently-writing file. I'd prefer not to alt-tab or split my tiny screen with two windows.

**Comment by wwfyde** [comment]
+1

**Comment by SzHeJason** [comment]
any update?

**Comment by Yesaye** [comment]
+1
Looking forward to this useful feature.

**Comment by indigofairyx** [comment]
++ YES Please! would be great to have files one side, outline on the other. jumping between them in project folders is unnecessarily cumbersome.

**Comment by Jia-py** [comment]
+1
